### DISCUSSION TITLE
Crontab is erroring out after 3.0.0

### DISCUSSION THREAD

**Discussion by rt-541** [mbround18/valheim-docker#851] [Help!] [answered]
![image](https://github.com/mbround18/valheim-docker/assets/8106052/3c449b90-1f80-45fc-a5ad-2b7eeceb3846)

```
      - AUTO_UPDATE=1
      - AUTO_UPDATE_SCHEDULE="0 4 * * *"
      - AUTO_BACKUP=1
      - AUTO_BACKUP_SCHEDULE="*/15 * * * *"
      - AUTO_BACKUP_REMOVE_OLD=1
      - AUTO_BACKUP_DAYS_TO_LIVE=3
      - AUTO_BACKUP_ON_UPDATE=1
      - AUTO_BACKUP_ON_SHUTDOWN=1
 ```
 when update or any crontab for that matter is enabled 
 
valheim-1  | 2024-05-28 20:45:38 - [Valheim][INFO]: Checking if cron directory and files exist...
valheim-1  | "-":0: bad minute
valheim-1  | errors in crontab file, can't install.

I think this this code block here 

https://github.com/mbround18/valheim-docker/blob/edc9f5d8830650886801059ac403a12a16303e80/src/scripts/entrypoint.sh#L189

this is how I'd fix it but I'm not ur supervisor 
```
# Apply cron job files
for file in /home/steam/cron.d/*; do
    echo "adding cronjob into crontab # <-- this should provide better logging in the future if someone puts in a bad crontab entry. 
    crontab $file 
done

# Start cron in the
cron -f & 
```

**Comment by rt-541** [comment]
Other notes beside this little hiccup great server and thank you!

**Reply by mbround18** [reply]
Thanks for the shell, ill toss it in an test :)

**Reply by mbround18** [reply]
Im testing this here :)

https://github.com/mbround18/valheim-docker/pull/857

**Reply by mbround18** [reply, marked as answer]
Fix in https://github.com/mbround18/valheim-docker/releases/tag/v3.0.1

**Reply by rt-541** [reply]
Thanks Looks good!
